dialects.md

```diff
@@ -514,26 +584,24 @@ The generated RDF graph will look like this:
 ]
 ```
 
-The rules for defining key-value nesting are the following:
-
-* Nodes with at least two properties can be used as ranges for properties defining key-value nesting. 
-
-* Mandatory properties in the range node must be defined as either `mapKey` or `mapValue`. Additional non-mandatory 
-properties will not be parsed.
-
-* `mapValue` can only be defined is `mapKey` is defined
-
-* Only properties with literal ranges can be defined as `mapKey`
-
-Failing to meet these requirements will result in violations
-
-Bear in mind the following considerations:
-
-* This change is merely syntactical; neither the parsed graph for the dialect instance nor the SHACL semantics for the 
-constraint will be affected by the change.
-
-* The range of the property defining the nesting will allow multiple objects in its value (defaults to 
-`allowMultiple: true`)
+#### Validation of nesting with value mapping
+Failing to meet the following conditions will raise a **violation** in the _dialect definition_:
+* Only property mappings which are `mandatory: true` can be set as _value-mapped property mapping_. 
+* Only property mappings which do not have union ranges can be set as _source property mapping_  <!-- TODO: this breaks compat with "Configuration" dialect -->
+* The `mapValue` and `mapTermValue` facets can only be defined if either `mapKey` or `mapTermKey` facets are defined (and valid) as well
+* The `mapValue` and `mapTermValue` facets are mutually exclusive and cannot be defined in conjunction
+* The value of the `mapValue` and `mapTermValue` facets should match some property mapping in the _range node mapping_ by either its name or property term
+* The value of the `mapValue` and `mapTermValue` facets cannot match the same property mapping as the `mapValue` and `mapTermValue` facets
```

Comment:
"s the `mapKey` and `mapTermKey`" facets quisite decir? tipo el mapValue obviamente tiene que ser un mapping distinto que el key?

Reply:
Exacto eso quize decir!